pkg/kubelet/eviction/eviction_manager.go

```diff
@@ -578,6 +587,18 @@ func (m *managerImpl) evictPod(pod *v1.Pod, gracePeriodOverride int64, evictMsg
 	m.recorder.AnnotatedEventf(pod, annotations, v1.EventTypeWarning, Reason, evictMsg)
 	// this is a blocking call and should only return when the pod and its containers are killed.
 	klog.V(3).InfoS("Evicting pod", "pod", klog.KObj(pod), "podUID", pod.UID, "message", evictMsg)
+	_, otelSpan := m.tracer.Start(ctx, "Eviction/EvictPod", trace.WithAttributes(
+		attribute.String("k8s.pod.uid", string(pod.UID)),
+		attribute.String("k8s.pod", klog.KObj(pod).String()),
+		attribute.String("k8s.pod.name", pod.Name),
+		attribute.String("k8s.namespace.name", pod.Namespace),
+	))
+	otelSpan.AddEvent("Evicting pod", trace.WithAttributes(
+		attribute.String("pod", klog.KObj(pod).String()),
+		attribute.String("podUID", string(pod.UID)),
+		attribute.String("message", evictMsg),
```

Comment:
even though these aren't part of the semantic conventions, we should namespace attribute keys.  Possibly k8s.eviction.message

Reply:
Done